<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="redLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Red</string>
    </property>
   </widget>
   <widget class="QLabel" name="greenLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Green</string>
    </property>
   </widget>
   <widget class="QLabel" name="blueLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="redEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="greenEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="blueEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QGraphicsView" name="colorView">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="updateButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Random</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QDial" name="redSlider">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QDial" name="greenSlider">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QDial" name="blueSlider">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>redEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>editUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>editUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>editUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sliderUpdate()</slot>
  <slot>editUpdate()</slot>
  <slot>buttonClicked()</slot>
 </slots>
</ui>
